lib/rspec/core/output_wrapper.rb

```diff
@@ -3,13 +3,17 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        @output.respond_to?(name, priv)
+      end
```

Comment:
How can `@output` be a string?  I thought we handled strings before wrapping it in this by opening the file and passing the file to this.

This class is specifically designed to wrap objects that implement Ruby's `IO` interface.  IMO, it's a bad idea to let it wrap any other kind of object (like a string).